OpenSim/Simulation/OpenSense/OpenSenseUtilities.cpp

```diff
@@ -360,4 +218,59 @@ OpenSenseUtilities::createOrientationsFileFromMarkers(const std::string& markers
     STOFileAdapter_<SimTK::Quaternion>::write(quaternions, fileName);
 
     return quaternions;
-}
\ No newline at end of file
+ }
+
+SimTK::Rotation OpenSenseUtilities::computeHeadingCorrection(
+        Model& model,
+            OpenSim::TimeSeriesTable_<SimTK::Quaternion_<double>>&
+                    quaternionsTable,
+            const std::string& baseImuName,
+            const SimTK::CoordinateDirection baseHeadingDirection) 
+{
+    SimTK::Rotation rotation;
+ 
+    // if a base imu is specified, perform heading correction, otherwise skip
+    if (!baseImuName.empty()) {
```

Comment:
Since the other functionality has been removed from this function, this is unnecessary. Perhaps now, if you use the function, you should have to give a valid baseImuName or it should throw an exception?